### ISSUE TITLE
TypeError: Object Variable has no method 'codePointAt'

### ISSUE THREAD

**Issue by denysdovhan** [firede/msee#32]
Hi there!

I use **msee** in my [**bash-handbook**](https://github.com/denysdovhan/bash-handbook). I've received [an issue report from user](https://github.com/denysdovhan/bash-handbook/issues/14) and it seems like a trouble with msee itself

Error stack:

```
/usr/lib/node_modules/bash-handbook/node_modules/msee/lib/msee.js:312
        var code = str.codePointAt(i);
                       ^
TypeError: Object Variable has no method 'codePointAt'
    at getStringWidth (/usr/lib/node_modules/bash-handbook/node_modules/msee/lib/msee.js:312:24)
    at /usr/lib/node_modules/bash-handbook/node_modules/msee/node_modules/text-table/index.js:33:21
    at Array.forEach (native)
    at forEach (/usr/lib/node_modules/bash-handbook/node_modules/msee/node_modules/text-table/index.js:73:31)
    at /usr/lib/node_modules/bash-handbook/node_modules/msee/node_modules/text-table/index.js:32:9
    at Array.reduce (native)
    at reduce (/usr/lib/node_modules/bash-handbook/node_modules/msee/node_modules/text-table/index.js:63:30)
    at module.exports (/usr/lib/node_modules/bash-handbook/node_modules/msee/node_modules/text-table/index.js:31:17)
    at tableFormat (/usr/lib/node_modules/bash-handbook/node_modules/msee/lib/msee.js:294:12)
    at processToken (/usr/lib/node_modules/bash-handbook/node_modules/msee/lib/msee.js:188:23)
```

**Comment by bucaran** [comment]
Perhaps @bluurn is using a previous msee version before this commit?

https://github.com/firede/msee/commit/1f154cd6fc1439a94fb08f1a68e53643ebcd81d8

Hey @martinheidegger, any thoughts?

**Comment by martinheidegger** [comment]
Note: I did publish `msee-preview` a preview variant of msee. @firede if you add me to the owners in npm I can also publish msee

**Comment by firede** [comment]
@martinheidegger done.

**Comment by martinheidegger** [comment]
Published 0.4.0 - please review :)
